api/src/main/java/dev/jsinco/brewery/util/Registry.java

```diff
@@ -21,11 +33,11 @@ public class Registry<T extends BreweryKeyed> {
     public static final Registry<CauldronType> CAULDRON_TYPE = fromEnums(CauldronType.class);
     public static final Registry<StructureMeta<?>> STRUCTURE_META = (Registry<StructureMeta<?>>) fromFields(StructureMeta.class);
     public static final Registry<StructureType> STRUCTURE_TYPE = (Registry<StructureType>) fromFields(StructureType.class);
-    public static final Registry<NamedDrunkEvent> DRUNK_EVENT = fromEnums(NamedDrunkEvent.class);
+    public static final Registry<NamedDrunkEvent> DRUNK_EVENT = fromClasses(ChickenNamedEvent.class, DrunkenWalkNamedEvent.class, DrunkMessageNamedEvent.class, FeverNamedEvent.class, HallucinationNamedEvent.class, KaboomNamedEvent.class, NauseaNamedEvent.class, PassOutNamedEvent.class, PukeNamedEvent.class, StumbleNamedEvent.class, TeleportNamedEvent.class);
```

Comment:
With my first suggestion, that code line could be this:
```suggestion
    public static final Registry<NamedDrunkEvent> DRUNK_EVENT = fromJson("/events/my_json_file.json", json -> NamedDrunkEvent.fromJson(json));
```
Or something like that, depends on the specifics.